docs/getting_started/tutorial.md

```diff
@@ -1,5 +1,9 @@
 # Your first Wagtail site
 
+This tutorial shows you how to build a blog using Wagtail. This tutorial gives you hands-on experience with some of Wagtail's features.
+
+It's recommended that you understand the fundamentals of Python programming and the use of the Django framework in order to easily follow this tutorial.
```

Comment:
How about

```suggestion
A basic knowledge of Python programming and the Django framework will help you follow this tutorial.
```

Reply:
I will seek a way to refactor this and avoid the use of the phrase "in order to." For the sake of  consistency, we agreed to adopt the Google developer documentation style guide. The style guide strongly advice against using "will" and it's past tense, "would". You can read more about this on the Google developer documentation style guide's [Word list](https://developers.google.com/style/word-list)